connector/MongoDBAtlasODBC.integration-test.query.pq

```diff
@@ -0,0 +1,250 @@
+      boolean = true, 
+      dbPointer = "{""$dbPointer"":{""$ref"":""namespace"",""$id"":{""$oid"":""000000000000000000000001""}}}", 
+      double = 14.96, 
+      integer = 100, 
+      javascript = "{""$code"":""function(){ }""}", 
+      javascriptWithScope = "{""$code"":""function(){ }"",""$scope"":{""foo"":""bar""}}", 
+      maxKey = "{""$maxKey"":1}", 
+      minKey = "{""$minKey"":1}", 
+      object = "{""foo"":""bar"",""objId"":{""$oid"":""000000000000000000000002""},""value"":3,""time"":{""$timestamp"":{""t"":200,""i"":0}}}", 
+      objectId = "{""$oid"":""000000000000000000000001""}", 
+      regularExpression = "{""$regularExpression"":{""pattern"":""a(bc)*"",""options"":""""}}", 
+      symbol = "{""$symbol"":""symbol""}", 
+      timestamp = "{""$timestamp"":{""t"":100,""i"":0}}" ] },
+
+    facts = 
+    {
+        Fact("Verify values in sales collection from navigation table",
+          Sales_Row,
+          Table.ToRecords(Table_List_Sales)
+        ),
+        Fact("Verify values in complex_types collection from navigation table",
+          Complex_Types_Row,
+          Table.ToRecords(Table_List_Complex_Types)
+        ),
+        Fact("Query matching database name works with unqualified collection name",
+          Sales_Row,          
+          Table.ToRecords(Query_Sales_Unqualified)
+        ),
+        Fact("Query different database name requires qualified collection name",
+          Complex_Types_Row,
+          Table.ToRecords(Query_Complex_Types_Qualified)
+        ),
+        Fact("Check schema, embedded complex types are tranformed to string",
+          {"Text.Type", "Logical.Type", "Text.Type", "Text.Type", "Text.Type", "DateTime.Type", "Text.Type"},
+          Table.Schema(Table_List_Sales)[TypeName]
+        ),
+        Fact("Check schema, complex types only are tranformed to string",
+          {"Text.Type", "Text.Type", "Logical.Type", "Text.Type", "Double.Type", "Int32.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type", "Text.Type"},
+          Table.Schema(Query_Complex_Types_Qualified)[TypeName]
+        ),
+        Fact("NativeTypeName facet is retained in complex types from query",
+          {"objectId", "array", "bool", "dbPointer", "double", "int", "javascript", "javascriptWithScope", "maxKey", "minKey", "object", "objectId", "regex", "symbol", "timestamp"},
+          Table.Schema(Query_Complex_Types_Qualified)[NativeTypeName]),
+
+        Fact("NativeTypeName facet is retained in complex types from navigation table",
+          {"objectId", "array", "bool", "dbPointer", "double", "int", "javascript", "javascriptWithScope", "maxKey", "minKey", "object", "objectId", "regex", "symbol", "timestamp"},
+          Table.Schema(Table_List_Complex_Types)[NativeTypeName])
+    },
+    report = Facts.Summarize(facts)
+][report];
+
+/// COMMON UNIT TESTING CODE - From DataConnectors/UnitTesting 
```

Comment:
This section is also in `MongoDBAtlasODBC.unit-test.query.pq`, I spent some time trying to put this in an common module file but was hitting issues.  Leaving it this way for now.